lib/master.js

```diff
@@ -177,14 +177,11 @@ class Master extends ClusterProcess {
      * @returns {Master} self
      *
      * @description Shortcut for:
-     *      master.getWorkersArray().forEach(function(worker) {
-     *          // fn
-     *      }, master);
+     *      master.getWorkersArray().forEach(fn);
      */
     forEach(fn) {
-        this.getWorkersArray().forEach(function(worker) {
-            fn.call(this, worker);
-        }, this);
+        this.getWorkersArray()
+            .forEach(fn);
```

Comment:
This changes previous behaviour and requires minor version bump at least.

Reply:
I'm planning to release this with major version bump and worker pools. Anyway, I've bumped minimal node version.